src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -1262,6 +1263,28 @@ public static Process Start(string fileName, string arguments)
             return Start(new ProcessStartInfo(fileName, arguments))!;
         }
 
+        /// <devdoc>
+        ///    <para>
+        ///       Starts a process resource by specifying the name of an
+        ///       application and a set of command line arguments. Associates the process resource
+        ///       with a new <see cref='System.Diagnostics.Process'/>
+        ///       component.
+        ///    </para>
+        /// </devdoc>
```

Comment:
devdoc shouldn't be used; this should just be a summary tag. The second sentence should probably be in a remarks tag, or deleted entirely. (I've not looked at the docs for process, but I'd suggest seeing what the existing docs say.)

Reply:
> devdoc shouldn't be used;

I saw that it's used in other `Start` overloads and assumed that it should be used (for some historical reason etc).

I am going to convert it to a summary tag, but should we do it for all other devdoc comments as well? Perhaps it would make the xml docs more friendly to docs automation? cc @carlossanlop

Reply:
We've not cleaned up the existing comments.  The shape of the new ones matter because they're automated to seed the docs.